aerogear-android-sdk-bom/pom.xml

```diff
@@ -39,6 +39,8 @@
         <!-- Auth -->
         <appauth.version>0.7.0</appauth.version>
         <jose4j.version>0.6.3</jose4j.version>
+        <customtabs.version>26.1.0</customtabs.version>
```

Comment:
I think we don't need a `customtabs.version` propoerties because it should be **always** equal `android.support.version`

Reply:
Ok cool, Ill remove that one now.